### PR TITLE
Umfrage Bugs

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/views/survey/currentSurveysConsortia.gsp [hunk 1]
```diff
@@ -349,7 +349,7 @@
                         </g:if>
                     </td>
                     <td>
-                        <g:link controller="survey" action="show" id="${surveyInfo?.id}" params="[surveyConfigID: surveyConfig.id]" class="ui button icon">
+                        <g:link controller="survey" action="show" id="${surveyInfo?.id}" params="[surveyConfigID: surveyConfig?.id]" class="ui button icon">
```
Comment (croqueGrec09) [inline app/grails-app/views/survey/currentSurveysConsortia.gsp:352]: Can that not be done by <g:if>?

Comment (Moetez16) [reply app/grails-app/views/survey/currentSurveysConsortia.gsp:352]: No.

Comment (klodav) [reply app/grails-app/views/survey/currentSurveysConsortia.gsp:352]: Link without target if surveyInfo / surveyConfig is null?

Comment (croqueGrec09) [reply app/grails-app/views/survey/currentSurveysConsortia.gsp:352]: That is what I remark as well!

**Pull request merged**